insights/misc/importer.py

```diff
@@ -7,11 +7,98 @@
 
 from misc.models import Artifact, Repo, Run, Token, Workflow
 
+class Log():
+    def __init__(self):
+        self.data = []
+
+    def append(self, string):
+        string = "%s: %s" % (datetime.datetime.now(), string)
+        print(string)
+        self.data.append(string)
+
+
+def get_workflows(log, repo, repo_from_api, k):
+    workflows = repo_from_api.get_workflows().get_page(k)
+    if not workflows:
+        log.append("No workflows for page: %s, stopping" % k)
+        return
+
+    log.append("Found workflows in page: %s" % k)
+    for workflow_from_api in workflows:
+        workflow = Workflow.objects.get_or_create(
+            workflow_id=workflow_from_api.id, repo=repo
+        )[0]
+        workflow.name = workflow_from_api.name
+        workflow.path = workflow_from_api.path.split("/")[-1]
+        workflow.save()
+        log.append("Saved workflow: %s" % workflow.path)
+
+    k += 1
```

Comment:
```
    k += 1
```

Consider adding some kind of cutoff - a max number of pages, stopping at workflows older than N months old, etc.

If nothing else it will speed up our testing on large repos.

Reply:
Oh good call, let's go for 100 or so.